### ISSUE TITLE
[Feature Request]Table显示外边框

### ISSUE THREAD

**Issue by easynet-cn** [view-design/ViewUI#86]
### What problem does this feature solve?
目前当border设置为false的时候，table两侧的外边框也不会显示，希望能提供属性配置，用来控制是否显示两侧外边框

### What does the proposed API look like?
无

**Comment by NNNNzs** [comment]
.ivu-table-wrapper {
    border: 1px solid #dcdee2;
}

**Comment by ELISHELL** [comment]
对 .ivu-table-wrapper  增加 border 后，假设父容器宽度为 100%、auto... 等，会导致table 的整体宽度无限增加

**Comment by shizhihuaxu** [comment]
Table 添加 border， 然后设置 th, td {border: none; }
